src/ts/mapboxMap.ts

```diff
@@ -195,18 +195,19 @@ module powerbi.extensibility.visual {
         }
 
         private validateOptions(options: VisualUpdateOptions) {
-            this.errorDiv.style.display = 'none';
-            this.errorDiv.innerText = '';
+            // Hide errorDiv
+            const element = this.errorDiv;
 
-            //check for GL Support
-            // if (!mapboxgl.supported()) {
-            //     this.errorDiv.innerHTML = Templates.noGlSupport
-            //     return false;
-            // }
+            // Hide div and remove any child elements
+            this.errorDiv.setAttribute("style", "display: none;");
+            while (element.hasChildNodes()) { 
+                element.removeChild(element.firstChild) 
+            }
```

Comment:
Would be easier to read if this assignment to `element` was closer to its use. Or just use `this.errorDiv` throughout since the while loop isn't in a separate function.